server/routes/api.js

```diff
@@ -23,13 +24,55 @@ module.exports = (services) => {
       throw err;
     }
   });
-
+  /* POST calculate */
   router.post('/calculate', (req, res) => {
     const dijkstra = new Dijkstra(req.body.from, req.body.dest, { MAX_DISTANCE: 4 });
     const result = dijkstra.solve();
     console.log(result);
     res.json(result);
   });
+  /* POST routes */
+  router.post('/routes', async (req, res) => {
+    try {
+      const routes = req.body;
+      const trip = {
+        tracknum: uuidv4(),
```

Comment:
Tracking number is never returned to the user. You should include this in the success response so the user can use it to track the package.

Reply:
@bill-feng Fixed it!